agent/exploits/webexploit.py

```diff
@@ -94,6 +103,16 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
                 vulnerabilities.append(vulnerability)
                 return vulnerabilities
 
+        if self.vuln_ranges is not None:
+            for pattern in version_pattern:
+                if (matched := pattern.search(resp.text)) is not None:
+                    extracted_version = matched.group()
```

Comment:
Isn't this a list?

Reply:
i also add support matching version if response have many